Upstream-Status: Inappropriate [embedded specific]

Don't build the doc examples - we don't need them and in fact they
never successfully built in previous iterations of the lttng-ust
recipe anyway.

Signed-off-by: Tom Zanussi <tom.zanussi@intel.com>

Index: lttng-ust-2.9.1/doc/Makefile.am
===================================================================
--- lttng-ust-2.9.1.orig/doc/Makefile.am
+++ lttng-ust-2.9.1/doc/Makefile.am
@@ -1,3 +1,3 @@
-SUBDIRS = . man examples
+SUBDIRS = . man
 
 dist_doc_DATA = java-agent.txt
